src/compose/ComposeBox.js

```diff
@@ -328,6 +338,11 @@ class ComposeBox extends PureComponent<Props, State> {
       ...this.inputMarginPadding,
       ...this.context.styles.backgroundColor,
     },
+    deactivatedGroupPMWarning: {
+      padding: 10,
+      backgroundColor: 'hsl(45, 100%, 51%)',
```

Comment:
Could you say more about how you arrived at these numbers? I wonder if there's a constant we can use for the `backgroundColor`, and if it might be slightly more readable to say `'black'` instead of `'hsl(0, 0%, 0%)'`.

Reply:
IIRC it was google material orange colour. But i've removed it now, since I realized that its not possible to send a message in a group PM where even one member has been deactivated.